app/routes/my-tickets/upcoming/list.js

```diff
@@ -97,6 +97,7 @@ export default class ListRoute extends Route {
       filter       : filterOptions,
       perPage      : 10,
       startingPage : 1,
+      sort         : 'event.starts-at',
```

Comment:
Shouldn't this be in controller

Reply:
I checked in the my-session it is added in this way